### PR TITLE
[INLONG-8360][Manager] Support previewing data of Kafka

### LINKED ISSUE #8360: [Improve][Manager] Support previewing data of Kafka
### Description

Support previewing data of Kafka.

The purpose of data preview is to observe the result of sending data in time after the user sends data, and take it out from kafka to verify the correctness of the data.

Please refer to the following pull requests:

- Pulsar
#8301

- Tube
#8357

- Dashboard
#8299
image

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by hnrainll**
### Prepare a Pull Request

- Fixes #8360 

### Motivation

Support previewing data of Kafka.

### Modifications

Support previewing data of Kafka.

### CODE REVIEW

**Comment by dockerzhang** [comment]
@hnrainll please add a UT for this PR. Thanks.

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/kafka/KafkaOperator.java [hunk 1]
```diff
@@ -79,4 +109,70 @@ public boolean topicIsExists(KafkaClusterInfo kafkaClusterInfo, String topic)
...
 
+    public List<BriefMQMessage> queryLatestMessage(KafkaClusterInfo kafkaClusterInfo, String topicName,
+            Integer messageCount, InlongStreamInfo streamInfo) {
+        LOGGER.info("begin to query message for topic {} in cluster: {}", topicName, kafkaClusterInfo);
+
+        Properties properties = new Properties();
+        properties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaClusterInfo.getUrl());
+        properties.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
+        properties.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
+        properties.put(ConsumerConfig.GROUP_ID_CONFIG, "QueryLatestMessage-" + Utils.getUUID());
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/kafka/KafkaOperator.java:120]: It's better to use the same subscription and reset the offset of the target topic to the latest for each query.

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/resource/queue/kafka/KafkaOperatorTest.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class KafkaOperatorTest extends ServiceBaseTest {
```
Comment (healchow) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/resource/queue/kafka/KafkaOperatorTest.java:49]: 1. `public` class.
2. Add javadoc for the class.

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/resource/queue/kafka/KafkaOperatorTest.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+    private static final String TopicName = "test";
```
Comment (healchow) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/resource/queue/kafka/KafkaOperatorTest.java:51]: For static and final varibles, please use BIG_CAMEL_CASE, thanks.

**Review by hnrainll — COMMENTED** [review]
he

**Review by healchow — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Pull request merged**